homeassistant/components/html5/__init__.py

```diff
@@ -1 +1,50 @@
 """The html5 component."""
+
+import logging
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import CONF_PLATFORM, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv, discovery
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DATA_HASS_CONFIG, DOMAIN
+from .issues import async_create_html5_issue
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM = Platform.NOTIFY
```

Comment:
This is not needed right?

Reply:
I think it is needed by the notify platform. There's some magic that happens and this appeared to be required or I wouldn't have added it.

Reply:
I don't know about any magic around PLATFORM

Reply:
Hmm I'll remove this and see if it still works. I don't know all of home assistant internals, but I do know there's a log of magic that happens with these constants.

Reply:
And if it is the case, please share what you find, I'm curious if that's the case

Reply:
Seems like this isn't needed. Removed.